omod/src/main/java/org/openmrs/module/reportingrest/web/controller/ReportingRestController.java

```diff
@@ -13,11 +13,40 @@
  */
 package org.openmrs.module.reportingrest.web.controller;
 
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.time.DateUtils;
+import org.apache.commons.logging.Log;
```

Comment:
This is our convention for logging: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging